docker/config/default.config.envsubst

```diff
@@ -9,8 +9,8 @@ network.host_ip=$RADIXDLT_HOST_IP_ADDRESS
 network.p2p.seed_nodes=$RADIXDLT_NETWORK_SEEDS_REMOTE
 network.p2p.use_proxy_protocol=$RADIXDLT_NETWORK_USE_PROXY_PROTOCOL
 
-state_hash_tree.gc.interval_sec=$RADIXDLT_STATE_TREE_GC_INTERVAL_SEC
-state_hash_tree.state_version_history_length=$RADIXDLT_STATE_TREE_STATE_VERSION_HISTORY_LENGTH
+state_hash_tree.gc.interval_sec=$RADIXDLT_STATE_HASH_TREE_GC_INTERVAL_SEC
```

Comment:
> making them match the properties

yeah, that's the reason.

So the properties were added quite some time ago, and they _can_ be used to minimize disk space usage, which makes me think they _are_ used by someone. I don't want to be rude to these people.
I could also argue that "property names not matching type names in deeper layers of code" is not a huge problem (because devops/node runners - who fiddle with properties daily - don't really care how we named related variables/types in the code, so to them, the naming is still consistent and they are glad it didn't change).

Reply:
Fair point :+1: